app/models/facility.rb

```diff
@@ -129,12 +129,12 @@ def problem_reservation_order_details
     complete_problem_order_details.reservations
   end
 
-  private
```

Comment:
Looks like it was only being used internally by `problem_non_reservation_order_details` and `problem_reservation_order_details` so there was no reason to make it public.